timemachine/cpp/src/tibd_exchange_move.cu

```diff
@@ -35,12 +37,12 @@ TIBDExchangeMove<RealType>::TIBDExchangeMove(
     : BDExchangeMove<RealType>(
           N, target_mols, params, temperature, nb_beta, cutoff, seed, proposals_per_move, interval),
       radius_(static_cast<RealType>(radius)), inner_volume_(static_cast<RealType>((4.0 / 3.0) * M_PI * pow(radius, 3))),
-      quaternion_offset_(0), d_rand_states_(DEFAULT_THREADS_PER_BLOCK), d_inner_mols_count_(1),
-      d_identify_indices_(this->num_target_mols_), d_partitioned_indices_(this->num_target_mols_),
-      d_temp_storage_buffer_(0), d_center_(3),
+      d_rand_states_(DEFAULT_THREADS_PER_BLOCK), d_inner_mols_count_(1), d_identify_indices_(this->num_target_mols_),
+      d_partitioned_indices_(this->num_target_mols_), d_temp_storage_buffer_(0), d_center_(3),
       d_uniform_noise_buffer_(round_up_even(NOISE_PER_STEP * this->RANDOM_BATCH_SIZE)), d_targeting_inner_vol_(1),
       d_ligand_idxs_(ligand_idxs), d_src_weights_(this->num_target_mols_), d_dest_weights_(this->num_target_mols_),
-      d_inner_flags_(this->num_target_mols_), d_box_volume_(1), p_inner_count_(1), p_targeting_inner_vol_(1) {
+      d_inner_flags_(this->num_target_mols_), d_box_volume_(1), p_inner_count_(1), p_targeting_inner_vol_(1),
+      d_translations_(round_up_even(TRANSLATIONS_PER_STEP * this->RANDOM_BATCH_SIZE)), d_selected_translation_(3) {
```

Comment:
+3?

Reply:
Is this the right allocation size? is `this->RANDOM_BATCH_SIZE=4`? where `(x,y,z,w)` is such that, xyz used for the translation, and w is the acceptance critera,